### DISCUSSION TITLE
search by analysis type mixes up samples by different contributors

### DISCUSSION THREAD

**Discussion by tilfischer** [ComPlat/chemotion_REPO#97] [Ideas]
Dear all,

starting on https://www.chemotion-repository.net/home/publications and searching "by analysis type:cyclic voltammetry (CV)" gives one result which is https://dx.doi.org/10.14272/UVLHGRADYISRGZ-UHFFFAOYSA-N.11 . 

Before this sample got published, the same search gave also just one result which was contributed by David Herrmann. If I search "by author:David Herrmann" these samples are shown:
![Screenshot 2024-03-27 134831](https://github.com/ComPlat/chemotion_REPO/assets/85482437/f3493a8a-bff1-47ab-a58b-259d0da7d17f)

Please note the sample to the very bottom on the right hand side. The Author displayed is "Alrik Dorn". If I click on that sample, I arrive at https://dx.doi.org/10.14272/UVLHGRADYISRGZ-UHFFFAOYSA-N.10 , which is the sample published by David but not the one by Alrik. 

The same can be observed if search for sample "by author:Niklas Oppel", while the result with "Alrik Dorn" as author leads to https://dx.doi.org/10.14272/UVLHGRADYISRGZ-UHFFFAOYSA-N.8 . 

Best,
Tillmann

**Comment by cllde8** [comment]
Dear @tilfischer,

Thank you for bringing this matter to our attention. Please refer to the video below where I utilized the names "David Herrmann," "Niklas Oppel," and "Alrik Dorn" as the search criteria for the author. Upon reviewing your screenshot (see image), I observed that the search criteria inputted was a "structure," thus affecting the search results accordingly.

[screencast-www.chemotion-repository.net-2024.03.28-13_49_04.webm](https://github.com/ComPlat/chemotion_REPO/assets/42910431/5fc471e3-517d-4265-b53e-e0b7f9d5e036)

![screenshot-github com-2024 03 28-13_50_00](https://github.com/ComPlat/chemotion_REPO/assets/42910431/4e410fcc-2d4c-458e-9fd4-b30f818e2b26)


Best regards,
Claire

**Comment by tilfischer** [comment]
Dear Claire,

Thank you for your reply. Searching by the names work i.e. I find the samples if I know the names of people who contributed samples with CV data. Also thank you for pointing that there was possibly still something added in the structure search. I started fresh and the search "by author:David Herrmann" gave one sample.

However, if I just search for "by analysis type:cyclic voltammetry (CV)" I only get one result, although there are more samples which have CV data. 

Best,
Tillmann

**Comment by cllde8** [comment]
Dear @tilfischer,

The "samples" page presents information from a molecule point of view, with each molecule displaying the latest associated sample publication. Users can easily access all associated sample publications by clicking on the molecule of interest. When searching for data on the "samples" page by a specified analysis type, such as cyclic voltammetry (CV), the page will display the molecule with the latest sample publication information. For example, in this case, it will be a sample published on 2024-03-12 by Alrik Dorn, and users can click on the record to access all sample publications.

Best regards,
Claire

**Comment by tilfischer** [comment]
Dear Claire,

Thank you for pointing this out. This was all but obvious to me. I added a small video, which shows one result for the search "by analysis type:cyclic voltammetry (CV)" with author "Alrik Dorn" (the most recent one) but shows more than one sample if this is selected i.e. the one by Nikklas, Claudia and David (and more of David not shown in the video):

https://github.com/ComPlat/chemotion_REPO/assets/85482437/e1093017-30d1-447a-b11b-fb5b54215848

@nicolejung: What is your opinion on that? I would have expected to have all samples published by someone listed on the pane to the left.

Best,
Tillmann

**Comment by cllde8** [comment]
Dear @tilfischer 

Yes, that is correct. We list molecules instead of samples in order to avoid presenting similar data to users in the list view, especially because the main information we want to provide to users in the list view is related to the molecular data (e.g., structure images, formulas, and names). As you can see, these take up a large portion of the list view record. While it is possible to list samples in the list view, the list view should be redesigned in my opinion. For example, if we search for "Analysis type: cyclic voltammetry (CV)" as per the current list view design, the list view will show 6 similar records.

The previous version of the "Samples" page only listed the molecules and their information (see image). This approach may be more straightforward for the user. 
![screenshot-www chemotion-repository net-2024 04 09-13_51_02](https://github.com/ComPlat/chemotion_REPO/assets/42910431/a2142fce-6d09-4c36-92b1-e0284c0c8b12)

In any case, I am glad you raised this topic for discussion. Would you mind if I move this to the "Discussion" thread and see others' inputs there?


Best regards,
Claire

**Comment by tilfischer** [comment]
Dear Claire,

I have no obligations to convert this to a GitHub discussion.

Best,
Tillmann

**Comment by cllde8** [comment]
Dear @tilfischer 

Thank you for your response. As discussed, since this ticket seems to be more of a feature discussion rather than a bug report, I will proceed to relocate it to the Discussion board, to get broader input and facilitate decision-making. Afterward, we'll create separate issues based on the outcomes of the discussion, if required.

Best regards,
Claire
